x/staking/client/cli/query.go

```diff
@@ -122,31 +123,31 @@ $ gaiacli query staking redelegations-from cosmosvaloper1gghjut3ccd8ay0zduzj64hw
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 
-			valAddr, err := sdk.ValAddressFromBech32(args[0])
+			valSrcAddr, err := sdk.ValAddressFromBech32(args[0])
 			if err != nil {
 				return err
 			}
 
-			bz, err := cdc.MarshalJSON(staking.NewQueryValidatorParams(valAddr))
+			bz, err := cdc.MarshalJSON(staking.QueryRedelegationParams{SrcValidatorAddr: valSrcAddr})
 			if err != nil {
 				return err
 			}
 
-			route := fmt.Sprintf("custom/%s/%s", storeKey, staking.QueryValidatorRedelegations)
+			route := fmt.Sprintf("custom/%s/%s", storeKey, staking.QueryRedelegations)
 			res, err := cliCtx.QueryWithData(route, bz)
 			if err != nil {
 				return err
 			}
 
-			var reds staking.Redelegations
-			cdc.MustUnmarshalJSON(res, &reds)
-			return cliCtx.PrintOutput(reds)
+			var resp staking.RedelegationResponses
+			cdc.MustUnmarshalJSON(res, &resp)
```

Comment:
In response to https://github.com/cosmos/cosmos-sdk/pull/4333/files#r283993574

Then I'd fix all these lines (there are many like this across clients code) - and this is definitely a more panic-prone instance.

Reply:
Indeed. Will update.